datasets/ph/most_wanted/ph_most_wanted.yml

```
@@ -0,0 +1,36 @@
+title: Top Most Wanted Persons in the Cordillera Regions
+entry_point: crawler.py
+prefix: ph
+coverage:
+  frequency: weekly
+  start: "2024-07-22"
+deploy:
+  schedule: "@weekly"
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+summary: >
+  The dataset contains information about individuals currently most wanted by the Philippine National Police in the Cordillera Region.
+description: |
+  The dataset contains detailed information about individuals wanted by the police authorities in the Cordillera Region. It includes 
+  various categories of criminals and terrorists, including communists and locals, who have rewards associated with their capture. 
+  The dataset includes each person's name, reward amount, offences, and case-control number (CC No.).
```

```suggestion
  The dataset contains detailed information about individuals wanted by the police authorities in the Cordillera Region. It includes 
  various categories of criminals and terrorists who have rewards associated with their capture. 
  The dataset includes each person's name, reward amount, offences, and case-control number (CC No.).
```

I'm not sure "communists and locals" adds much, while it read a bit like communists are criminals, which isn't quite what they intend with the heading "PRO COR communist terrorist group (CTG)"